### PR TITLE
adds CM barricades and sprites

### PR DESCRIPTION
**Pull request by 2cwldys**
adds CM's deployable barricades, and sprites for FOBs (Forward Operating Bases) established on enemy ship hallways or future planets.

### CODE REVIEW

File: code/game/barricade.dm [hunk 1]
```diff
@@ -0,0 +1,980 @@
...
+	if(closed)
+		return TRUE
+
+	if(mover && mover.throwing)
+		if(is_wired && iscarbon(mover)) //Leaping mob against barbed wire fails
+			if(get_dir(loc, target) & dir)
+				return FALSE
+		return TRUE
+
+	if(istype(mover, /obj/vehicle))
```
Comment (Kmc2000) [inline code/game/barricade.dm]: This needs removing, it's referring to a vehicle var that we don't have

File: code/game/barricade.dm [hunk 2]
```diff
@@ -0,0 +1,980 @@
...
+		return FALSE
+	else
+		return TRUE
+
+/obj/structure/barricade/attack_animal(mob/user as mob)
+	return attack_alien(user)
+
+/obj/structure/barricade/attack_alien(mob/living/carbon/xenomorph/M)
+	M.do_attack_animation(src)
+	obj_integrity -= rand(M.xeno_caste.melee_damage_lower, M.xeno_caste.melee_damage_upper)
```
Comment (Kmc2000) [inline code/game/barricade.dm]: obj_integrity -= rand(M.melee_damage_lower,M.melee_damage_upper)

File: code/game/barricade.dm [hunk 3]
```diff
@@ -0,0 +1,980 @@
...
+/*----------------------*/
+// PLASTEEL
+/*----------------------*/
+
+/obj/structure/barricade/plasteel
+	name = "plasteel barricade"
+	desc = "A very sturdy barricade made out of plasteel panels, the pinnacle of strongpoints. Use a blowtorch to repair. Can be flipped down to create a path."
+	icon_state = "plasteel_closed_0"
+	max_integrity = 600
+	crusher_resistant = TRUE
```
Comment (Kmc2000) [inline code/game/barricade.dm]: We don't have crushers, so best remove this and anything it refers to

File: code/game/barricade.dm [hunk 4]
```diff
@@ -0,0 +1,980 @@
...
+		obj_integrity -= 50
+
+	update_health()
+	return TRUE
+
+/*----------------------*/
+// SANDBAGS
+/*----------------------*/
+
+/obj/structure/barricade/sandbags
```
Comment (Kmc2000) [inline code/game/barricade.dm]: Please remove this one, /tg/ already has sandbags

Comment (2cwldys) [reply code/game/barricade.dm]: I think I have that now in change commit 746e6d4

File: code/game/barricade.dm (hunk 3 above)
Comment (2cwldys) [reply code/game/barricade.dm]: I think I have that now in change commit 746e6d4

File: code/game/barricade.dm (hunk 2 above)
Comment (2cwldys) [reply code/game/barricade.dm]: Oh yeah that'd make sense, no xenos

I think I have that now in change commit 746e6d4

File: code/game/barricade.dm (hunk 1 above)
Comment (2cwldys) [reply code/game/barricade.dm]: I think I have that now in change commit 746e6d4

**Comment by 2cwldys** [comment]
https://pastebin.com/9G80rrFj

Errors at compile

**Comment by Kmc2000** [comment]
This ended up being stupidly fiddly, but it's here in:
https://github.com/DDMers/sephora/pull/8

**Pull request closed without merging**